pages/Config-Packs.md

```diff
@@ -1,25 +1,58 @@
-Config packs define a Terra world generator. They are YAML-based, and define all the necessary components to create a
+#### What exactly is a config pack?
+
+Config packs are a group of configuration files define a Terra world generator. These configuration files are [YAML](https://en.wikipedia.org/wiki/YAML)
+based, and define all the necessary components to create a
 complete world generator with Terra. Users may install multiple config packs simultaneously, allowing the use of
-multiple Terra-based generators on the same server.  
-Config packs occupy the directory `plugins/Terra/packs/`. 
-
-## Installing a Config Pack
-Installing a config pack is easy! First, download your config pack, and place it in the `plugins/Terra/packs/`
-directory. Then, open `bukkit.yml` (found in the root directory of your server), and add the following lines to the
-end of the file:
-```yaml
-worlds:
-  world_name:
-    generator: Terra:CONFIG_ID
+multiple Terra-based generators on the same server, with any number of worlds configured to use any of those generators.
+
+#### Where can I find config packs?
+
+You can find a list of community made Terra configuration packs on our [Community Packs](./Community-Packs) page.
+
+#### How do I install a pack?
+
+Config packs can be installed similarly to resource packs, which is a fairly straight forward process. First, download
+your config pack, and place it in the
+directory `plugins/Terra/packs/`, this is where all Terra config packs will be located, including the `DEFAULT` pack
+which is automatically installed with Terra.
+
+> NOTE - A pack can be either contained inside a simple folder, or within an archive such as a .zip file.
```

Comment:
I just assumed any archive would work since terra tried to load a jar as a pack once lol
![image](https://user-images.githubusercontent.com/73215501/110395378-1f2d3800-80c2-11eb-82ce-cec73608723e.png)
will make these changes

Reply:
JAR loading is being removed in 5.0, the original plan was to have addons be in the same JAR as config packs, hence JAR being a valid pack extension.    
Since we aren't doing that, JARs are no longer considered valid packs in 5.0 and up.